client/extensions/woocommerce/woocommerce-services/views/shipping-label/label-purchase-modal/customs-step/style.scss

```diff
@@ -0,0 +1,79 @@
+.customs-step__package-container {
+	margin-bottom: 16px;
+}
+
+.customs-step__package-container:last-child {
+	margin-bottom: 0;
+}
+
+.customs-step__package-name {
+	font-size: 16px;
+	font-weight: 600;
+	margin-bottom: 8px;
+}
+
+.customs-step__restrictions-row {
+	margin-top: 8px;
```

Comment:
Done in https://github.com/Automattic/wp-calypso/pull/24974/commits/edfa0f3f5c5c827a09fc7d74387af926badb0362 (used `16px` since I seem to remember we're using a "multiples of 8px" grid).